packages/design/src/colors.css

```diff
@@ -4,9 +4,20 @@
   --color-job: var(--color-yellowGreen);
   --color-invoice: var(--color-purple);
 
+  --color-interactive: var(--color-green);
+  --color-learning: var(--color-lightBlue);
+  --color-destructive: var(--color-red);
+  --color-disabled: var(--color-grey);
+  --color-disabled--secondary: var(--color-grey--lighter);
   --color-outline: var(--color-yellow--light);
   --color-error: var(--color-red);
 
+  --color-text: var(--color-greyBlue--dark);
```

Comment:
Should I add `--color-heading` here? We just pass "blue" to all of the headers in `Heading`